saleor/graphql/attribute/utils.py

```diff
@@ -57,9 +57,15 @@ class AttributeAssignmentMixin:
     be unable to build or might only be partially built.
     """
 
-    REFERENCE_VALUE_NAME_MAPPING = {AttributeEntityType.PAGE: "title"}
-
-    ENTITY_TYPE_TO_MODEL_MAPPING = {AttributeEntityType.PAGE: page_models.Page}
+    REFERENCE_VALUE_NAME_MAPPING = {
+        AttributeEntityType.PAGE: "title",
+        AttributeEntityType.PRODUCT: "name",
+    }
+
+    ENTITY_TYPE_TO_MODEL_MAPPING = {
+        AttributeEntityType.PAGE: page_models.Page,
+        AttributeEntityType.PRODUCT: product_models.Product,
+    }
```

Comment:
@maarcingebala  Maybe we should store this mapping in `saleor/attribute/__init__.py`?

Reply:
Good idea, they will be closer to AttributeEntityType and it will be easier to add new entity types later (no need to jump between files).

Reply:
I cannot move here `ENTITY_TYPE_TO_MODEL_MAPPING`, when models are imported I got the message: `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet`. And it's not caused by circular imports.

Reply:
We decided to left it as it is.